CHANGELOG_PENDING.md

```diff
@@ -2,6 +2,18 @@
 
 ## v0.34.21
 
+Release highlights include:
+- A new RPC configuration flag, `discard_abci_responses`, which, if enabled,
+  discards all ABCI responses except the latest one in order to reduce disk
+  space usage in the state store. Note that, if enabled, this means that the
```

Comment:
omit "Note that" and "this means that" 

"When enabled" is more clear than "if enabled"